src/lib/openapi/standard-responses.ts

```diff
@@ -0,0 +1,23 @@
+import { OpenAPIV3 } from 'openapi-types';
+
+export const unauthorizedResponse: OpenAPIV3.ResponseObject = {
+    description:
+        'Authorization information is missing or invalid. Provide a valid API token as the `authorization` header, e.g. `authorization:*.*.my-admin-token`.',
+} as const;
+
+const standardResponses = {
+    401: unauthorizedResponse,
+} as const;
```

Comment:
Could remove the `as const`s.

Reply:
Can do for sure, but I'm curious as to what  the reasoning is for it? Not that I'm opposed in any way, I just want to know why.

Reply:
The `as const` tells the compiler to infer the narrowest possible type for the object. 

But this narrow type is lost when assigning the value to a less specific variable (`OpenAPIV3.ResponseObject`).

Reply:
Alright, that's fair enough. Could perhaps type it as a partial response object to prevent you from adding anything that the response object doesn't have? But I'm happy to leave it without.